src/components/shell.css

```diff
@@ -58,66 +60,75 @@
   height: var(--shell-header-height);
   box-sizing: border-box;
   position: fixed;
-  /* left: calc(50vw - var(--shell-control-icon-width) - var(--shell-control-icon-width) / 2); */
-  left: 10vw;
-  width: 80vw;
```

Comment:
Moved to `@media (--wide-screen)` section, and modified to be `left:50vw, width:50vw`